CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
 # Braintree Android SDK Release Notes
 
 ## unreleased
-
+* PayPal
+  * Add PayPal App Switch vault flow (BETA)
+    * Add `enablePayPalAppSwitch` property to `PayPalVaultRequest` for App Switch support
+    * Require `PayPalVaultRequest.userAuthenticationEmail` for App Switch support
+    * Require `PayPalClient.appLinkReturnUrl` for App Switch support
+    * Send `link_type` and `paypal_installed` in `event_params` when available to PayPal's analytics service (FPTI)
```

Comment:
Is this something we can omit from the changelog? I think this would be an internal change.

Reply:
yeah 🤔 , im basically following the same approach we used for [iOS](https://github.com/braintree/braintree_ios/pull/1335/commits/ec3488a72c332087048470f408268314077d78dc). In fact, we’ve even updated the CHANGELOG with internal changes in the past. I’m open to making adjustments if we agree as a team on omitting these types of changes.

Reply:
Gotcha! we can merge this in as is and modify it if the team decides to only include external changes.